chat-agent/tests/test_content_moderation.py

```
@@ -0,0 +1,836 @@
+        
+        is_flagged, category = await check_moderation("Can you help me create a guest WiFi account?")
+        assert is_flagged is False
+        assert category == ""
+
+
+@pytest.mark.unit
+@pytest.mark.security
+class TestAPIErrorHandling:
+    """Test API error handling."""
+
+    async def test_network_error_handling(self, mock_openai_client, mocker: MockerFixture):
```

Ruff (0.15.10)

[error] Undefined name `MockerFixture`

(F821)